proto/buf/alpha/registry/v1alpha1/webhook.proto

```diff
@@ -17,26 +17,82 @@ syntax = "proto3";
 package buf.alpha.registry.v1alpha1;
 
 // WebhookService exposes the functionality for a caller to
-// subscribe to or unsubscribe to a Webhook for a given repository.
+// create/delete/list Webhooks for a given repository.
 service WebhookService {
-  // SubscribeToRepository for subscribing to a specific repository.
-  rpc SubscribeToRepository(SubscribeToRepositoryRequest) returns (SubscribeToRepositoryResponse) {}
-  // UnsubscribeToRepository for unsubscribing to a specific repository.
-  rpc UnsubscribeToRepository(UnsubscribeToRepositoryRequest) returns (UnsubscribeToRepositoryResponse) {}
+  // Create a webhook.
+  rpc CreateWebhook(CreateWebhookRequest) returns (CreateWebhookResponse) {}
+  // Delete a webhook.
+  rpc DeleteWebhook(DeleteWebhookRequest) returns (DeleteWebhookResponse) {}
+  // Lists the webhooks for a given repository.
+  rpc ListWebhooks(ListWebhooksRequest) returns (ListWebhooksResponse) {}
 }
 
-// SubscribeToRepositoryRequest is the proto request representation of a
-// webhook request body to subscribe.
-message SubscribeToRepositoryRequest {}
+// CreateWebhookRequest is the proto request representation of a
+// webhook request body.
+message CreateWebhookRequest {
+  // The events to subscribe to for the given repository.
+  repeated WebhookEvent event = 1;
+  // The owner name of the repository in the corresponding subscription request.
+  string owner_name = 2;
+  // The repository name that the subscriber wishes create a subscription for.
+  string repository_name = 3;
+  // The subscriber's callback URL where notifications
+  // should be delivered. It is considered good practice
+  // to use a unique callback URL for each subscription.
```

Comment:
`It is considered good practice
  // to use a unique callback URL for each subscription.`

I'm generally of the opinion that usage guidelines like this shouldn't live in the protobuf definitions - particularly without more context (why?). But I'm not sure what are our current practices about those, and I don't feel strongly about it.